app/test/shared/markdown_test.dart

```diff
@@ -255,6 +255,56 @@ void main() {
     });
   });
 
+  group('<summary>', () {
+    test('FIXME: <summary> must not exists in a list item', () {
+      expect(
+        markdownToHtml('- a<summary>b\n- c'),
+        '<ul>\n'
+        '<li>a<summary>b</summary></li>\n'
+        '<li>c</li>\n'
+        '</ul>\n',
+      );
+    });
```

Comment:
```suggestion
  testMarkdown(String name, String md, html) {
    test(name, () {
      // TODO: Normalize HTML before comparing
      expect(markdownToHtml(md), html));
    });
  }

    testMarkdown('FIXME: <summary> must not exists in a list item', 
       '- a<summary>b\n- c',
        '<ul>\n'
        '<li>a<summary>b</summary></li>\n'
        '<li>c</li>\n'
        '</ul>\n',
    );
```

Or even better maybe, improve our golden test framework and make it more generically reusable.